specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2023-03-01-preview/Capabilities.json

```diff
@@ -0,0 +1,562 @@
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "fastProvisioningSupportedEnum",
+            "modelAsString": true
+          }
+        },
+        "supportedFastProvisioningEditions": {
+          "type": "array",
+          "description": "List of supported server editions for fast provisioning",
+          "items": {
+            "$ref": "#/definitions/FastProvisioningEditionCapability"
+          },
+          "x-ms-identifiers": [
+            "supportedSku"
+          ],
+          "readOnly": true
+        },
+        "geoBackupSupported": {
+          "type": "string",
+          "description": "Determines if geo-backup is supported in this region.",
+          "readOnly": true,
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "geoBackupSupportedEnum",
+            "modelAsString": true
+          }
+        },
+        "zoneRedundantHaSupported": {
+          "type": "string",
+          "description": "A value indicating whether Zone Redundant HA is supported in this region.",
+          "readOnly": true,
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "zoneRedundantHaSupportedEnum",
+            "modelAsString": true
+          }
+        },
+        "zoneRedundantHaAndGeoBackupSupported": {
+          "type": "string",
+          "description": "A value indicating whether Zone Redundant HA and Geo-backup is supported in this region.",
+          "readOnly": true,
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "zoneRedundantHaAndGeoBackupSupportedEnum",
+            "modelAsString": true
+          }
+        },
+        "storageAutoGrowthSupported": {
+          "type": "string",
+          "description": "A value indicating whether storage auto-grow is supported in this region.",
+          "readOnly": true,
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "storageAutoGrowthSupportedEnum",
+            "modelAsString": true
+          }
+        },
+        "onlineResizeSupported": {
+          "type": "string",
+          "description": "A value indicating whether online resize is supported in this region for the given subscription.",
+          "readOnly": true,
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "onlineResizeSupportedEnum",
+            "modelAsString": true
+          }
+        },
+        "restricted": {
+          "type": "string",
+          "description": "A value indicating whether this region is restricted.",
+          "readOnly": true,
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
```

Comment:
Why not call the values 'Restricted' and 'Unrestricted' then?

Reply:
Updated description.